back/app/controllers/omniauth_callback_controller.rb

```diff
@@ -204,6 +204,8 @@ def selected_locale(omniauth_params)
     selected_locale = omniauth_params['sso_pathname'].split('/', 2)[1].split('/')[0]
 
     return selected_locale if selected_locale != locales.first && locales.include?(selected_locale)
+
+    return locales.first
```

Comment:
Assume this will get superceded by your other PR

Reply:
Yeah, I'll remove this change and merge the other PR to the feature branch